### PR TITLE
Job  link

### PR DESCRIPTION
**Pull request by achrafbsibiss**

### CODE REVIEW

File: app/views/job_applications/_form.html.erb [hunk 1]
```diff
@@ -66,7 +66,7 @@
       </div>
 
       <div class="col-span-full">
-          <%= form.label :resume, t("attributes.job_application.attach_a_resume") %>
+          <%= form.label :resume, t("attributes.job_application.attach_a_receipt") %>
```
Comment (mohamedrez) [inline app/views/job_applications/_form.html.erb:69]: why is it a receipt ?

**Review by mohamedrez — CHANGES_REQUESTED** [review]

File: app/views/job_applications/_form.html.erb (hunk 1 above)
Comment (achrafbsibiss) [reply app/views/job_applications/_form.html.erb:69]: it's just by fault, sorry

**Review by mohamedrez — APPROVED** [review]

File: Gemfile [hunk 2]
```diff
@@ -90,6 +90,7 @@ group :test do
   gem "selenium-webdriver", "4.10.0"
   gem "webdrivers"
   gem "database_cleaner"
+  gem "rails-controller-testing"
```
Comment (mohamedrez) [inline Gemfile:93]: why we had to add this new gem ?

**Review by mohamedrez — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (achrafbsibiss) [reply Gemfile:93]: this  i do the  "bundle exec rspec ./spec/requests/job_application_spec.rb"  he can check for the layout

**Review by itsAndL — APPROVED** [review]

File: app/controllers/job_applications_controller.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 class JobApplicationsController < ApplicationController
+  layout "custom_layout", only: %i[new create]
+  skip_before_action :authenticate_user!, only: %i[new create]
   before_action :set_job_application, only: %i[infos show edit update destroy delete_resume update_aasm_state]
-  before_action :set_breadcrumbs, only: %i[index show]
+  before_action :set_breadcrumbs, only: :index
```
Comment (itsAndL) [inline app/controllers/job_applications_controller.rb:5]: need to get it back to the previous one

**Review by itsAndL — CHANGES_REQUESTED** [review]
get things back that will not make an effect

**Pull request closed without merging**